server/src/main/java/io/crate/expression/InputFactory.java

```diff
@@ -238,7 +240,13 @@ public Input<?> visitReference(Reference ref, Void context) {
             }
             implementation = referenceResolver.getImplementation(ref);
             if (implementation == null) {
-                throw new IllegalArgumentException("Column implementation not found for: " + ref);
+                String schema = ref.ident().tableIdent().schema();
+                String msg = SysSchemaInfo.NAME.equals(schema) ||
+                    InformationSchemaInfo.NAME.equals(schema) ?
```

Comment:
not sure if we should do it for information_schema.

Reply:
Probably should be `Schemas.READ_ONLY_SYSTEM_SCHEMAS.contains` - as it can happen for any system table